src/lib/lwan.h

```diff
@@ -377,6 +377,7 @@ struct lwan_request {
     struct lwan_value url;
     struct lwan_value original_url;
     struct lwan_connection *conn;
+    struct lwan_strbuf *headers;
```

Comment:
Please mark this one as `const`:

```suggestion
    const struct lwan_strbuf *const headers;
```

So that it's not possible to change it.

Also, would be nice if it could be renamed to something else; `headers` can be easily confused with the request headers from the client. Maybe `response_headers` or something like that?